src/components/toolbar/Toolbar.tsx

```diff
@@ -251,13 +252,6 @@ Toolbar.PopoverItem = function ToolbarPopoverItem(
       interactionKind={popoverInteractionKind}
       hasBackdrop={popoverInteractionKind === 'click'}
       hoverCloseDelay={0}
-      targetProps={{
-        style: {
-          fontSize: '1.125em',
-          width: 'fit-content',
-          height: 'fit-content',
-        },
-      }}
```

Comment:
Then this was useless? I wonder why it was there in the first place.

Reply:
I think I just forgot to remove it when I changed to implementation to use `renderTarget`

[fix: ensure popover toolbar item does not grow](https://github.com/zakodium-oss/react-science/commit/9d8a0b459c813295af29cf90200e35540b495840)